keyboards/xelus/valor/frl_tkl/readme.md

```diff
@@ -0,0 +1,13 @@
+# Valor FRL TKL
+
```

Comment:
Do you have a pic about the PCB and the board by any chance?

Reply:
![image](https://user-images.githubusercontent.com/17491233/95039713-494a8a80-071d-11eb-9eaf-8d659b794430.png)

I only have a KLE currenlty.